test/Bridges/Constraint/interval.jl

```diff
@@ -456,6 +456,18 @@ function test_runtests()
         """,
         eltype = Rational{Int},
     )
+    MOI.Bridges.runtests(
```

Comment:
I don't think I've looked at `Bridges.runtests` recently. This is an amazingly clear test given the complexity of what's being tested.

Reply:
There was a period of weeks/months, where I felt like all I was doing was tidying up tests and refactoring things. Twas a long slog, but now it's trivial to write new bridges and I'm a lot more confident that the bridge code is correct.